src/t_stream.c

```diff
@@ -1986,7 +1986,7 @@ void streamRewriteTrimArgument(client *c, stream *s, int trim_strategy, int idx)
     decrRefCount(arg);
 }
 
-/* XADD key [(MAXLEN [~|=] <count> | MINID [~|=] <id>) [LIMIT <entries>]] [NOMKSTREAM] <ID or *> [field value] [field value] ... */
+/* XADD key [NOMKSTREAM] [(MAXLEN [~|=] <count> | MINID [~|=] <id>) [LIMIT <entries>]]  <ID or *> field value [field value...] */
```

Comment:
unless i'm missing something, i still argue that this change isn't needed.
it's not that this change is harmful, but i don't wanna open up the door for an endless chase to exactly match the formal docs and code comments.
as long as the code comments are not incorrect or misleading, they're ok IMHO.